src/main/java/seedu/address/model/Model.java

```diff
@@ -86,7 +86,7 @@
     ObservableList<Assignment> getFilteredAssignmentList();
 
     /** Returns an unmodifiable view of the filtered assignment list */
-    List<Assignment> getFilteredAssignmentList(Name name);
+    List<Assignment> getPersonAssignmentList(Name name);
```

Comment:
Perhaps it will be clearer to leave the name as ```getFilteredAssignmentList``` since it is initialise with a filtered list?

Reply:
I think it's fine if the method name are the same but in different class if they are returning the same type of object. This may help us know that they are similar and can help keep code maintainable

Reply:
So I should overload the method? As they have different parameter requirements

Reply:
Yeap you are spot on the methods